### DISCUSSION TITLE
Kotlin Support

### DISCUSSION THREAD

**Discussion by charlieduzstuf** [fgsoftware1/fgengine#7] [Ideas]
https://github.com/JetBrains/kotlin

**Comment by fgsoftware1** [comment]
> https://github.com/JetBrains/kotlin

it won't workout

**Reply by charlieduzstuf** [reply]
It's a new language that metaverse uses, that's why u might not understand it

**Reply by fgsoftware1** [reply]
> It's a new language that metaverse uses, that's why u might not understand it

i know what kotlin. it's the main lang for android btw. the prob will be JVM.

**Reply by charlieduzstuf** [reply]
Cool

**Reply by fgsoftware1** [reply]
i'll lock this for now
